mmv1/products/healthcare/FhirStore.yaml

```diff
@@ -48,16 +48,8 @@ examples:
       policyChanged:
         ' acctest.BootstrapPSARoles(t, "service-", "gcp-sa-healthcare",
         []string{"roles/bigquery.dataEditor", "roles/bigquery.jobUser"})'
-  - !ruby/object:Provider::Terraform::Examples
-    name: 'healthcare_fhir_store_notification_config'
```

Comment:
Lets keep this test around to make sure we don't accidentally break the deprecated field. We can hide it from the docs if you don't want to show it to users